types/experiments.ts

```diff
@@ -58,6 +58,10 @@ export interface NimbusExperiment {
   /** A list of probe set slugs relevant to the experiment analysis */
   probeSets: Array<string>;
 
+  /** A list of featureIds the experiment contains configurations for.
+   */
+  featureIds?: Array<string>;
```

Comment:
Optional because this is non-breaking, I will make this non-optional (empty array) in a future breaking change